di_website/templates/contactus/includes/contact_details.html

```diff
@@ -0,0 +1,57 @@
+{% load wagtailcore_tags %}
+
+<section class="section">
+    <div class="row row--narrow">
+        <h2 class="section__heading">Where we are based</h2>
```

Comment:
Would be best if these are added dynamically ... The country locations may not change any time soon, but the addresses could... Also, this may be reused somewhere.